pyvista/examples/downloads.py

```diff
@@ -639,6 +639,22 @@ def download_vtk_logo():
     return _download_and_read("vtk.png", texture=True)
 
 
+def download_sky_box_cube_map():
+    """Download a skybox cube map texture."""
+    sets = ['posx', 'negx', 'posy', 'negy', 'posz', 'negz']
+    images = ['skybox2-' + suffix + '.jpg' for suffix in sets]
+    texture = pyvista.Texture()
+    texture.cube_map = True  # Must be set prior to setting images
+    for i, fn in enumerate(images):
+        image = _download_and_read(fn)
+        flip = _vtk.vtkImageFlip()
+        flip.SetInputDataObject(image)
+        flip.SetFilteredAxis(1)  # flip y axis
+        flip.Update()
+        texture.SetInputDataObject(i, flip.GetOutput())
```

Comment:
Would there be a sane way to make this more _pyvistonic_ (:stuck_out_tongue:)? Looking at `Texture.cube_map`, am I reading it right that one has to do this 6-element `SetInputDataObject` dance to get the cube map working with a custom texture?

Reply:
Done.